solidity/dashboard/src/components/LiquidityRewardCard.jsx

```diff
@@ -143,17 +154,39 @@ const LiquidityRewardCard = ({
             &nbsp;to fetch the the total pool value and KEEP token in USD.
           </Tooltip>
           <h2 className={"liquidity__info-tile__title text-mint-100"}>
-            {formattedApy}
+            {formattedApy.value === Infinity ? (
+              <span>&#8734;</span>
+            ) : (
+              <>
+                {formattedApy.prefix}
+                <CountUp
+                  end={formattedApy.value}
+                  // Save previously ended number to start every new animation from it.
+                  preserveValue
+                  decimals={2}
+                  duration={1}
+                  formattingFn={formattingFn}
+                />
+              </>
+            )}
           </h2>
           <h6>Estimate of pool apy</h6>
         </div>
         <div className={"liquidity__info-tile bg-mint-10"}>
           {isFetching ? (
             <Skeleton tag="h2" shining color="color-grey-60" />
           ) : (
-            <h2
-              className={"liquidity__info-tile__title text-mint-100"}
-            >{`${formattedPercentageOfTotalPool}%`}</h2>
+            <h2 className={"liquidity__info-tile__title text-mint-100"}>
+              {formattedPercentageOfTotalPool.prefix}
```

Comment:
We could also remove this if we are planning to do the same things for percentageOfTotalPool as for APY (so keeping the value only in `formattedPercentageOfTotalPool` and modifying formattingFn.